fairroot/base/sim/FairMCApplication.cxx

```diff
@@ -55,24 +55,21 @@
 #include <TObject.h>            // for TObject
 #include <TParticlePDG.h>       // for TParticlePDG
 #include <TROOT.h>              // for TROOT, gROOT
-#include <TRefArray.h>          // for TRefArray
 #include <TSystem.h>            // for TSystem, gSystem
 #include <TVirtualMC.h>         // for TVirtualMC
 #include <TVirtualMCStack.h>    // for TVirtualMCStack
-class TParticle;
+#include <float.h>              // for DBL_MAX
```

Comment:
Shouldn't we use the C++ headers <cfloat> and <cstdlib> instead of the ones from C?

Reply:
Thanks for finding this one!

Fixed.